Sources/ElasticSwift/ElasticSwift.swift

```diff
@@ -16,40 +16,44 @@ public class RestClient: ESClient {
         self.init(settings: Settings.default)
     }
     
-    public func makeGet<T: Codable>() -> GetRequestBuilder<T> {
-        return GetRequestBuilder<T>(withClient: self)
+    public func makeGet<T: Codable>(fromIndex index: String, id: String) -> GetRequestBuilder<T> {
```

Comment:
Same as previous comment on mandatory fields.
I.e. the GET require the INDEX and the ID, so they are now not optional and constructors require them.
Same for other Builders and Requests
BTW as per your comment in the previous PR, in this way I've removed the unwanted "validation" method on Builders

Reply:
This current implementation of the `build()` is very simplistic. However, to put rules and guards around the required fields and value to generate correct request `build()` is the place to put that in place.